src/components/app/list-new/AppList.tsx

```diff
@@ -35,15 +35,15 @@ import { getAppList } from '../service'
 import moment from 'moment'
 import { getUserRole } from '../../userGroups/userGroup.service'
 
-export default function AppList({isSuperAdmin, appListCount} : AppListPropType) {
+export default function AppList({ isSuperAdmin, appListCount }: AppListPropType) {
     const location = useLocation()
     const history = useHistory()
     const params = useParams<{ appType: string }>()
     const { serverMode, setPageOverflowEnabled } = useContext(mainContext)
     const [dataStateType, setDataStateType] = useState(AppListViewType.LOADING)
     const [errorResponseCode, setErrorResponseCode] = useState(0)
-    const [lastDataSyncTimeString, setLastDataSyncTimeString] = useState('')
-    const [lastDataSync, setLastDataSync] = useState(false)
+    const [lastDataSyncTimeString, setLastDataSyncTimeString] = useState<React.ReactNode>('')
```

Comment:
why are we using `React.ReactNode`? it can simply be of `string` type

Reply:
It is not. If you will check renderDataSyncingText that's a jsx component, In `syncing` text required some CSS on the top of it.